src/components/ActionButton/ActionButton.tsx

```diff
@@ -10,6 +10,11 @@ import type { ClassName, PropsWithSpread } from "types";
 export const LOADER_MIN_DURATION = 400; // minimium duration (ms) loader displays
 export const SUCCESS_DURATION = 2000; // duration (ms) success tick is displayed
 
+export enum Label {
+  Waiting = "Waiting for action to complete",
```

Comment:
Why TitleCase? Shouldn't it be UPPERCASE (like consts)?

Reply:
I think we've been switching to that format because it's how a lot of the examples in TypeScript are written: https://www.typescriptlang.org/docs/handbook/enums.html, though the docs use both formats: https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-compile-time.

I've seen style guides recommend both formats. Maybe @petermakowski has more thoughts on this as I think he instigated the switch?

Reply:
Hmm… seems TypeScript community haven't decided yet: https://github.com/xojs/eslint-config-xo-typescript/issues/43

Reply:
Do we use enums elsewhere in our code? I guess we should just be consistent, the format doesn't matter too much.

Reply:
We use enums in maas-ui for labels (although I'm personally not convinced they're a better choice than regular JS objects).